source/views/components/filter/section-list.js

```diff
@@ -22,10 +22,10 @@ export function ListSection({filter, onChange}: PropsType) {
 	function buttonPushed(tappedValue: ListItemSpecType) {
 		let result
 
-		if (includes(selected, tappedValue)) {
+		if (some(selected, tappedValue)) {
 			// if the user has tapped an item, and it's already in the list of
 			// things they've tapped, we want to _remove_ it from that list.
-			result = without(selected, tappedValue)
+			result = reject(selected, tappedValue)
```

Comment:
This should still be "without".

Reply:
Okay actually

Reply:
My initial comment here was wrong. Because you're assigning to a value, you should use the non-mutating "reject" here, _but_ it needs a callback as the second argument. 

`reject(arr, item => item === tappedValue)` should be what you're looking for.

Reply:
<img width="531" alt="screen shot 2018-03-31 at 9 55 46 pm" src="https://user-images.githubusercontent.com/24640726/38169375-5194aa72-352e-11e8-81d9-f7e4af4a840e.png">
I was using the `matches` shorthand from the lodash documentation, which I tested and it works just fine..if you want me to change it for readability purposes I can

Reply:
Ah. Well. That makes sense. I would like you to change it away from the matches() shorthand, though; if we're going to use reject and some, let's use the Array.prototype versions instead - `selected.reject(item => _.isDeepEqual(item, tappedValue))`, and the same for the "some" function - "array.some". 

Lodash v4, when it came out, started moving away from the matches shorthands in some of its functions, and I feel like it'll keep removing them in v5. Plus, we don't use them anywhere else in the app, so it's more consistent to use the explicit callbacks here.